src/utils/withTheme.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import { theme } from '../theme';
+
+const mergeTheme = parentTheme => ({ ...theme, ...parentTheme });
```

Comment:
doing shallow merge for now, we can think of deep merge.

Reply:
shallow merge is enough, if we are not going to add class names.